Gherkin2Mtm/Gherkin2Mtm/merge.bat

```diff
@@ -14,6 +14,6 @@ git commit -m "Changes made by the Gherkin2Mtm tool ***NO_CI***"
 ECHO GIT STATUS
 git status
 ECHO GIT PUSH %sourceBranch%
-git push origin HEAD:%BUILD_SOURCEBRANCH%
+git push --force origin HEAD:%BUILD_SOURCEBRANCH%
```

Comment:
I'm not a big fan of `--force`, since it can have some side-effects when working with multiple people on the same branch.
I don't know this can happen in this case, but I'm more fan of using the `--force-with-lease` option. Force-with-lease has an extra check on the remote before overwriting the branch.

See: https://stackoverflow.com/questions/52823692/git-push-force-with-lease-vs-force

> `force` overwrites a remote branch with your local branch.
> `--force-with-lease` is a safer option that will not overwrite any work on the remote branch if more commits were added to the remote branch (by another team-member or coworker or what have you). It ensures you do not overwrite someone elses work by force pushing.

I always use `--force-with-lease`, but I'm not sure if this works in this use-case.

Reply:
let me try to use that and see if that works